.changeset/empty-swans-drive.md

```diff
@@ -0,0 +1,6 @@
+---
+'@roadiehq/backstage-plugin-wiz': patch
+'@roadiehq/plugin-wiz-backend': patch
+---
+
+Add links for issues, order by created date
```

Comment:
This is probably a minor not a patch as it changes functionality?

Reply:
Yeah, good shout